controllers/tmf-apis/ordering.js

```diff
@@ -517,7 +517,10 @@ var ordering = (function(){
 
                         if (Object.keys(ordering).length == 1 && 'orderItem' in ordering) {
                             updateItemsState(req, ordering, previousOrdering, false, callback);
+                        } else if ('note' in ordering) {
```

Comment:
You cannot do this in this way, you have to check that the provider is only trying to update the notes. With this solution, if the provider includes a note field he will be able to modify the whole order